test/metabase/pulse/pulse_integration_test.clj

```diff
@@ -559,3 +559,62 @@
+       data-tables))))
+
+(deftest number-viz-shows-correct-value
+  (testing "Static Viz. Render of 'Number' Visualization shows the correct column's first value #32362."
+    (mt/dataset test-data
+      (let [ ;; test card 1 'narrows' the query to a single column (the 'tax' field)
+            test-card1 {:visualization_settings {:scalar.field "TAX"}
+                        :display                :scalar
+                        :dataset_query          {:database (mt/id)
+                                                 :type     :query
+                                                 :query    {:source-table (mt/id :orders)
+                                                            :fields       [[:field (mt/id :orders :tax) {:base-type :type/Float}]]}}}
+            ;; test card 2's query returns all cols/rows of the table, but still has 'Number' viz settings (scalar.field "TAX")
+            test-card2 {:visualization_settings {:scalar.field "TAX"}
+                        :display                :scalar
+                        :dataset_query          {:database (mt/id)
+                                                 :type     :query
+                                                 :query    {:source-table (mt/id :orders)}}}]
+        (mt/with-temp [Card          {card-id1 :id} test-card1
+                       Card          {card-id2 :id} test-card2
+                       Dashboard     {dash-id :id} {:name "just dash"}
+                       DashboardCard {dash-card-id1 :id} {:dashboard_id dash-id
+                                                          :card_id      card-id1}
+                       DashboardCard {dash-card-id2 :id} {:dashboard_id dash-id
+                                                          :card_id      card-id2}
+                       Pulse         {pulse-id :id :as pulse} {:name         "Test Pulse"
+                                                               :dashboard_id dash-id}
+                       PulseCard             _             {:pulse_id          pulse-id
+                                                            :card_id           card-id1
+                                                            :dashboard_card_id dash-card-id1}
+                       PulseCard             _             {:pulse_id          pulse-id
+                                                            :card_id           card-id2
+                                                            :dashboard_card_id dash-card-id2}
+                       PulseChannel {pulse-channel-id :id} {:channel_type :email
+                                                            :pulse_id     pulse-id
+                                                            :enabled      true}
+                       PulseChannelRecipient _ {:pulse_channel_id pulse-channel-id
+                                                :user_id          (mt/user->id :rasta)}]
+          (is (= ["2.07" "2.07"]
```

Comment:
Does each of these values correspond to the scalar returned for each card? Might want to add a comment to that effect if so.